floor_height_node:
    ros__parameters:
        engineering:
            - "Normal"
            - "Concerte"
        option:
            - "StoreyHeight"
        
        I_R_C: [0.0750241, 0.0155045, 0.997061, -0.997164, 0.0071819, 0.0749201, -0.0059992, -0.999854, 0.0159993]          
        laserLineVectorInCamera: [-0.180453, -0.0155347, -0.983461]
        laserLineOriginPointInCamera: [-181.27, 51.9314, 47.2844]

        servoMotorRotationAxisVectorInCamera: [0.983986, -0.0515143, -0.170298]
        servoMotorOriginPointInCamera: [4.83526, 158.359, -20.0337]

        machineHeight: 1051.2739


       